tests/expectations/metrics/test_core.py

```diff
@@ -524,6 +524,94 @@ def test_quantiles_metric_spark(spark_session):
     assert results == {desired_metric.id: [1.0, 2.0, 3.0]}
 
 
+def test_column_histogram_metric_pd():
+    engine = build_pandas_engine(pd.DataFrame({"a": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]}))
```

Comment:
and here?

Reply:
@Shinnnyshinshin Same -- style -- we would do it throughout...